web-app/src/components/DataTable/index.jsx

```diff
@@ -59,15 +62,17 @@ class DataTable extends Component {
   };
 
   render() {
-    const { data, columns, emptyMessage } = this.props;
+    const { data, columns, emptyMessage, loading } = this.props;
     const formattedColumns = this.buildColumns(columns);
 
     const labels = formattedColumns.reduce((acc, column) => {
       acc.push(column.Header);
       return acc;
     }, []);
 
-    if (data.length > 0) {
+    if (data.length < 0 && !loading) {
+      return <p>{emptyMessage}</p>;
```

Comment:
emptyMessage?

Reply:
It’s the message that’s displayed when there’s no data. May get rid of it honestly and just show an empty table instead.

Reply:
Ah I see, yeah whatever you think is better. For a second I was wondering whether it was just empty that variable haha.

Reply:
I'll keep it like this for now just so I can work on more MVP critical issues.